### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by micaeljtoliveira**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -124,3 +126,49 @@ double-precision topography file.
...
+
+```console
+module use /g/data/ik11/spack/modules
+module load domain-tools
+```
+
+If instead you wish to build and install the tools from the sources, you can
+follow a slighly modified version of above step-be-step instructions:
+```console
+module load intel-compilers netcdf
```
Comment (aekiss) [inline README.md]: should be
```
module load intel-compiler netcdf
```

**Comment by aekiss** [comment]
looks ok apart from that one typo

File: README.md (hunk 1 above)
Comment (micaeljtoliveira) [reply README.md]: Good catch! Should be fixed now.

**Review by aekiss — APPROVED** [review]
lgtm

**Pull request merged**